src/axom/inlet/examples/example1_expected_output

```diff
@@ -0,0 +1,16 @@
+thermal_solver/mesh/filename: found /data/star.mesh

```

Comment:
I am pretty sure @kennyweiss  was more interested in the sphinx output of the example.  Also can you add it as a comment of the PR so we can look at it rather than a checked in file.